drivers/spi/spi_nxp_lpspi/spi_nxp_lpspi_common.c

```diff
@@ -133,6 +133,7 @@ int spi_nxp_init_common(const struct device *dev)
 	const struct spi_mcux_config *config = dev->config;
 	struct spi_mcux_data *data = dev->data;
 	int err = 0;
+	LPSPI_Type *base = (LPSPI_Type *)DEVICE_MMIO_NAMED_GET(dev, reg_base);
```

Comment:
nit: follow reverse christmas tree style (put this at the top since it is the longest)